### PR TITLE
Add apriltags library and orogen component.

### PR DESCRIPTION
**Pull request by dbcesar**
Add apriltags component to the rock-core package_set. 
The library is directlty downloaded from the developer web page and the orogen component is already in the rock-perception repository.

### CODE REVIEW

**Comment by goldhoorn** [comment]
Please add a maintainer tag to the manifest.xml

**Comment by dbcesar** [comment]
Done. Anything else? Thanks!

**Comment by dbcesar** [comment]
Hi, @goldhoorn. Is it ok now?

**Comment by goldhoorn** [comment]
Semms fine for me but I have this week no time for testing can someone else do a compile check?

**Comment by dbcesar** [comment]
@doudou, could you please check it?

File: patches/apriltags.patch [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
++target_link_libraries(apriltags pthread m)
+diff -urN ../apriltag-2015-03-18/manifest.xml ./manifest.xml
+--- ../apriltag-2015-03-18/manifest.xml	1970-01-01 01:00:00.000000000 +0100
++++ ./manifest.xml	2015-04-30 17:20:35.639306306 +0200
+@@ -0,0 +1,12 @@
++<package>
++  <description brief="Apriltags Marker Detection, source file can be find in http://april.eecs.umich.edu/wiki/index.php/AprilTags">
++  
++  </description>
++  <author>April Robotics Laboratory, Diego Cesar/diego.cesar@dfki.de</author>
```
Comment (goldhoorn) [inline patches/apriltags.patch]: Sorry don't saw this before:  are you really the author of the package?, i assume this is a 3th parity project?

**Comment by dbcesar** [comment]
@goldhoorn Sorry, a cut/paste mistake. Now it's ok!

**Comment by goldhoorn** [comment]
I will merge now, please next time squash your commits

**Pull request merged**